admin_api.go

```diff
@@ -213,18 +215,20 @@ func (a *AdminAPI) UnsubscribeWebhook(ctx context.Context, cmd *commands.CancelW
 	return nil
 }
 
-// NewAdminAPIWithXPriv initializes a new AdminAPI instance using an extended private key (xPriv).
-// This function configures the API client with the provided configuration and uses the xPriv key for authentication.
-// If any step fails, an appropriate error is returned.
+// UTXOs fetches a paginated list of UTXOs via the Admin XPubs API.
+// The response includes UTXOs along with pagination details, such as page number,
+// sort order, and sorting field.
 //
-// Note: Requests made with this instance will be securely signed.
-func NewAdminAPIWithXPriv(cfg config.Config, xPriv string) (*AdminAPI, error) {
-	authenticator, err := auth.NewXprivAuthenticator(xPriv)
+// Optional query parameters can be applied using the provided query options.
+// The response is unmarshaled into a *queries.UtxosPage struct.
```

Comment:
![image](https://github.com/user-attachments/assets/1f5e6812-e153-45f5-96e0-8d641fcafaa1)

Sorry that I previously corrected you with the "double L".

As far as we are consistent it's ok 🙈